.gitlab-ci.yml

```diff
@@ -124,6 +137,24 @@ default:
     # this job runs only on nightly pipeline with the mentioned variable, against `master` branch
     - if: $CI_COMMIT_REF_NAME == "master" && $CI_PIPELINE_SOURCE == "schedule" && $PIPELINE == "nightly"
 
+.merge-ref-into-master:            &merge-ref-into-master
+  before_script:
+      - git fetch origin +master:master
+      - git fetch origin +$CI_COMMIT_REF_NAME:$CI_COMMIT_REF_NAME
+      - git checkout master
+      - git config user.email "ci@gitlab.parity.io"
+      - git merge $CI_COMMIT_REF_NAME --verbose --no-edit
+
+.cargo-check-benches-script:       &cargo-check-benches-script
+  script:
+    - mkdir -p artifacts/benches/$CI_COMMIT_REF_NAME
+    - SKIP_WASM_BUILD=1 time cargo +nightly check --benches --all
+    - 'cargo run --release -p node-bench -- ::node::import::native::sr25519::transfer_keep_alive::paritydb::small --json
```

Comment:
quotes here to avoid gitlab issues with colons?

Reply:
exactly